apps/prairielearn/src/models/questions.ts

```diff
@@ -0,0 +1,24 @@
+import sqldb = require('@prairielearn/postgres');
+import AnsiUp from 'ansi_up';
+
+const ansiUp = new AnsiUp();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+export async function getQuestions(course_id, course_instances) {
+  const params = {
+    course_id: course_id,
+  };
+  const result = await sqldb.queryAsync(sql.questions, params);
```

Comment:
This should use `queryRows` and include a Zod schema.

By convention, we no longer separate out `params` into a separate variable; just inline the object in the function call.

Reply:
@nwalters512 do we have a list somewhere of the "new and improved" query functions? It's basically `queryRow`, `queryOptionalRow`, and `queryRows`, with everything else being deprecated?

Reply:
Can I pass on queryRows() for now? this is existing code, and the schema is rather complex, including containing assessments, tags, and other things that types are defined for

Reply:
Ok, so I just went ahead and figured out all the proper typing for this, so its all in there now. But I want you know that it wasn't very fun :)